src/Compilers/Core/Portable/Symbols/IEventSymbol.cs

```diff
@@ -58,5 +57,14 @@ public interface IEventSymbol : ISymbol
         /// Properties imported from metadata can explicitly implement more than one event.
         /// </remarks>
         ImmutableArray<IEventSymbol> ExplicitInterfaceImplementations { get; }
+
+        /// <summary>
+        /// Returns the parameters of this event.  Parameters on events are available 
+        /// only in VisualBasic and not in C#.  This property will only return parameters
+        /// if they were directly declared on the Event symbol.  i.e. an event of the form:
+        /// "Public Event E(sender As Object)".  For any other type of event, an empty array
+        /// will be returned.
```

Comment:
It looks like returning an empty array in this case is ambiguous. One cannot tell if there are no parameters, or event form is wrong.